docs/modules/general/proto.mdx

```diff
@@ -0,0 +1,286 @@
+- `<allow>` and `<deny>` can now be used anywhere a filter is expected, and actually function how they were supposed to i.e. they cause the filter to be ignored (skipped over) if it does not match.
+- Blitz titles are no longer a part of the [Blitz Module](/docs/modules/objectives/blitz), instead they are set using the map sub-element `<game>` and can be used with any gamemode.
+- Match time limit is no longer part of the `<score>` or `<blitz>` module, instead it is defined directly in the root element.
+
+  - ```xml
+    <map proto="1.4.0">
+    <time result="objectives">5m</time>
+    <!-- Other modules. -->
+    </map>
+    ```
+
+  ***
+
+  <span class="badge badge--primary">Change</span>
+  <br />
+
+- The top-level tags `<filters>` and `<regions>` are now the same thing. You can define filters, regions, and `<apply>`s in either one (remember that regions are a type of filter, have been for a while now).
+- Any filter tag anywhere can have an `id` attribute.
+- The `<apply>` tag accepts both references and inline definitions for its `region`, `kit`, and all of its filter properties, e.g. the two tags below are equivalent:
+
+  - ```xml
+    <apply region="effect-pad" kit="effect-kit" filter="only-red" block="only-leaves" />
+    <apply>
+        <region>
+            <cuboid min="1,2,3" max="4,5,6"/> <!-- effect-pad -->
+        </region>
+        <kit>
+            <potion>...</potion> <!-- effect-kit -->
+        </kit>
+        <filter>
+            <team>red-team</team>
+        </filter>
+        <block>
+            <material>leaves</material>
+        </block>
+    </apply>
+    ```
+
+- The `safe` and `spread` attributes of spawns can now be combined.
+- A items slot attribute is no longer required in kits. This attribute now also accepts Mojang slot names e.g. `slot.hotbar.8` or `armor.head`.
+  ***
+
+<span className="badge badge--success">New</span>
+<br />
+
+- There are two built-in regions called `everywhere` and `nowhere` that contain all points and no points, respectively.
+  - All of the new built-ins have equivalent tags of the same name i.e. `<everywhere/>` & `<nowhere/>`.
+- A team can have a minimum required size with the `min=""` attribute.
+- Objectives have a `required` attribute that specifies if they are required to complete the match.
+  ***
+
+<span className="badge badge--info">Removed</span>
+<br />
+
+- The `<multitrade/>` tag has been removed, it is now always enabled.
+
+### Changes in 1.3.6
+
+<span className="badge badge--danger">Breaking</span>
+<br />
+
+- Regions for cores, destroyables, wools, portals & score boxes must be defined in a `<region>` sub-element or referenced in a `region` attribute.
+
+  - ```xml
+    <core>
+        <region>
+            <cuboid min="..." max="..."/>
+        </region>
+    </core>
+    <core region="my-region"/>
+    ```
+  - Similarly, spawn regions must be defined in a `<regions>` sub-element or referenced in a `regions` attribute.
```

Comment:
This can stay as a sub-bullet, but make the XML example in the same bullet as the text.